### ISSUE TITLE
Error trying to slice coco dataset using CLI

### ISSUE THREAD

**Issue by kalfasyan** [obss/sahi#452]
I tried to slice my image dataset using your CLI:
```
sahi coco slice --image_dir data/ --dataset_json_path coco_labels/COCO_dataset.json --slice_size 512 --overlap_ratio 0.2 --ignore_negative_samples
```
I get the error: ```AssertionError: you must provide a bbox or polygon``` from coco.py l:188. 
Is that coming from an image that doesn't have a bounding box and isn't that a "negative sample" that should be ignored? Removing the `--ignore_negative_samples` didn't seem to have an effect.

**Comment by kalfasyan** [comment]
I saw it was solved in [fix annotation dict without segmentation #374
](https://github.com/obss/sahi/pull/374#issue-1124688769) and I had an older version. Closing this issue.

**Comment by fcakyon** [comment]
@kalfasyan good to hear that! Feel free to open issues if you face another problems 👍
